meshapp/src/components/login-form.tsx

```diff
@@ -22,6 +25,52 @@ const LoginScreen = (props: ComponentProps) => {
     console.log(formData);
   };
 
+  const navigate = useNavigate();
+  const [cookies, setCookie] = useCookies(['user_id']);
+
+  useEffect(() =>{},[cookies]);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const response = await axiosInstance.post('accounts/login/',{
+      "email":formData.user,
+      "password":formData.pass
+    })
+      .then((axiosResponse) => {
+        if(axiosResponse.data.enabled_2fa){
+          setCookie('user_id', axiosResponse.data.user_id, { path:'/', maxAge:150 });
+          axiosInstance.post('accounts/set-two-factor-auth/', {
+            "accountID":axiosResponse.data.user_id
+          })
+          .then(() => {
+            navigate('/otp');
+          })
+          .catch((error) => {
+            alert(error.response.data.message);
+          });
+        }
+        else{
+          navigate('/logged_in_home')
+        }
+      })
+      .catch((error) => {
+        if(error.response){
+          // The request was made and the server responded with a status code
+          // that falls out of the range of 2xx
+          alert(error.response.data.message); //TODO: Improve the error response 
```

Comment:
Can you make an issue for this TODO

Reply:
I was going to make an issue for this but I thought since it wasn't that complicated I went ahead and added sweetalert so now it looks like this for errors:
![image](https://github.com/LetsMesh/Site/assets/71865513/f59e69ba-f652-4264-acb2-cb86db2852e1)
And this is just a quick pop-up when the user is verified via OTP
![image](https://github.com/LetsMesh/Site/assets/71865513/8b909e95-9b64-4156-89a9-6b33c8252960)

[5a261af](https://github.com/LetsMesh/Site/pull/316/commits/5a261af9968c62e4f505966aa3493a49409b2916)

Reply:
Can you remove the comment for the TODO if you're satisfied with how you solved it otherwise yea make an issue for later addressing (or closing) don't want to have todo statements in code without issues attached

Reply:
yeah I believe it should be removed on the latest commit